crates/ironrdp-client/src/main.rs

```diff
@@ -69,6 +79,9 @@ fn main() -> anyhow::Result<()> {
         rt.block_on(client.run());
     });
 
+    #[cfg(windows)]
+    std::mem::forget(win_clipboard);
```

Comment:
question: Do we really need to forget the `win_clipboard`? I think it already lives until `gui.run` terminates, because it is in the scope of the main function.

Reply:
right, it shouldn't be necessary now

Reply:
actually it's needed, as nothing else holds it otherwise and the variable isn't read.

Reply:
Oh, it's just for the warning?
You can prefix the variable name with _ to tell the compiler that it's intentionally not used and the warning will go away

Reply:
"error: collection is never read"(https://github.com/Devolutions/IronRDP/actions/runs/7519920189/job/20469012292). /me getting bored :)

Reply:
I see! Sounds like a clippy false-positive. I don’t think `forget`ting the variable is the proper way to silent it, but it will not create any problem either in this case.